docs/dataformats/index.rst

```diff
@@ -19,6 +19,16 @@ we simply use general-purpose file formats define our own semantics as described
 in the :ref:`dataformats_file_formats` section. 
 
 
+.. _time_gammapy:
+
+***************
+Time in Gammapy
+***************
+
+Gammapy adopts the exact same time standard as the Fermi Science Tools as described on this page: `Cicerone: Data - Time in Fermi Data Analysis <http://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Cicerone/Cicerone_Data/Time_in_ScienceTools.html>`_.
```

Comment:
I thought you wanted to have it there?
 https://github.com/gammapy/gammapy/pull/278#discussion_r32211328
Please make a suggestion where should it go.